config/application.rb

```diff
@@ -77,9 +77,14 @@ class Application < Rails::Application
 
     # Show the GDPR "I accept" cookies banner by default.
     # This setting is overridden in test.rb,
-    # so the tests don't have to click "I accept".)
+    # so the tests don't have to click "I accept".
     config.hide_accept_cookies_banner = false
 
+    # Show the survey sparrow survey by default.
+    # This setting is overridden in test.rb,
+    # so the tests don't to fill out the survey.
+    config.hide_survey_sparrow = false
```

Comment:
You're putting this in Rails `config` instead of our `ScihistDigicoll::Env`?   If we use `Env`, we can turn it on/off from heroku config vars, without having to redeploy code to turn it on/off. 

I feel like the variable should probably be `show_survey_sparrow` instead of `hide_survey_sparrow`, reverse the meaning. 

The code as is seems to assume that as long as this code is in the repo, we will be showing surveys, and to stop showing surveys we actually need to remove it from the repo.  Is that a safe assumption, or do we actually need config to turn them on and off?  Because we might want to turn them on and off at specific times that aren't the same times we deploy;  because we might leave the code in for future surveys?

Reply:
Good idea -- yeah, let's move this to an ENV var.

Reply:
Also reversed the name of the var and its behavior to `show_survey_sparrow`  -with a default value of "false". Anything other than a value of "true" in the env var will suppress the survey.